### ISSUE TITLE
Property 'show_in_taskbar' doesn't work on nw v0.14.15

### ISSUE THREAD

**Issue by asinverse** [nwjs/nw.js#4898]
I update the nw version in my application from v0.12.3 to v0.14.15, I've read the changes and followed them. But, I got this error:

> Invalid value for argument 2. Property 'show_in_taskbar': Unexpected property.
> It works well in v0.12.3 (at least in my side), and no change in the documentation.

However, `win.setShowInTaskbar(false)` works well.

Thank you.

**Comment by Klaster1** [comment]
This is probably caused by [`show_in_taskbar` missing from schema](https://github.com/nwjs/nw.js/blob/c0533b0f04130111b1173e2bfeb12e6762475533/src/api/nw_window.idl#L17). The [Window API has a check in place](https://github.com/nwjs/nw.js/blob/c0533b0f04130111b1173e2bfeb12e6762475533/src/resources/api_nw_window.js#L618), so hopefully this can be easily fixed.

**Comment by fritx** [comment]
```plain
Error: Invalid value for argument 2. Property 'toolbar': Unexpected property, Property 'show_in_taskbar': Unexpected property.
    at validate (extensions::schemaUtils:34:13)
    at Object.normalizeArgumentsAndValidate (extensions::schemaUtils:117:3)
    at Object.<anonymous> (extensions::binding:377:30)
```

Any progress?

**Comment by ghostoy** [comment]
Should be merged to #4970 and a PR is available at #5455.
